views/index-copy.ejs

```diff
@@ -0,0 +1,125 @@
+                <div class="animated-icon1">
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </div>
+            </button>
+
+            <!-- Collapsible content -->
+            <div class="collapse navbar-collapse" id="navbarSupportedContent20">
+
+                <!-- Links -->
+                <ul class="nav navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link" href="#carousel-slides">Quotes<span class="sr-only">(current)</span></a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#form">Add</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#about">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#footer">Contact</a>
+                    </li>
+                </ul>
+                <!-- Links -->
+            </div>
+            <!-- Collapsible content -->
+        </div>
+    </nav>
+    <!-- End of Navbar -->
+
+
+    <video autoplay muted loop id="wall-video">
+      <source src="images/wall.mp4" type="video/mp4">
+      Your browser does not support HTML5 video.
+    </video>
+
+
+    <!-- Start of Carousel of Quotes -->
+    <div id="carousel-container" class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <div id="carousel-example-generic" class="carousel slide" data-ride="carousel">
+                    <!-- Indicators -->
+                    <ol class="carousel-indicators">
+                        <li data-target="#carousel-example-generic" data-slide-to="0" class="active"></li>
+                        <% for (let j = 1; j < users.length && j < 5; j++) { %>
+                            <li data-target="#carousel-example-generic" data-slide-to="<% j %>"></li>
+                        <% } %>
+                    </ol>
+
+                    <!-- Wrapper for slides -->
+                    <div id="carousel-slides" class="carousel-inner" role="listbox">
+                        <div class="item active quotes center-block text-justified">
+                            <h1><%= users[users.length-1].quote %></h1>
+                            <p><%= users[users.length-1].name %></p>
+                            <br>
+                            <div>
+                                <img class="heartEmpty pull-right" src="images/heartEmpty.png" alt="like?"/>
+                                <img class="heartRed pull-right" src="images/heartRed.png" alt="like!"/>
+                            </div>
+                        </div>
+                        <% for (let i = users.length-2; i >= 0 && i >= users.length-5; i--) { %>
+                            <div class="item quotes center-block text-justified">
```

Comment:
What is the purpose of using `users.length-2` and `users.length-5` here?  The intention is not clear.